app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.java

```diff
@@ -17,15 +17,15 @@
     LoginPresenter() {
     }
 
-    public void setPreferences(Context context, String emailField) {
+    void setPreferences(Context context, String emailField) {
```

Comment:
I think that shared preferences can be inject, so you dont need to pass context to presenter to call shared preferences. We can talk about it together.